app/bundles/CoreBundle/Entity/CommonRepository.php

```diff
@@ -223,6 +224,22 @@ public function detachEntity($entity)
         $this->getEntityManager()->detach($entity);
     }
 
+    public function refetchEntity(object &$entity): void
```

Comment:
Can we return the entity rather than mutating it? And even if we wouldn't return it, the `&` sign is unnecessary for objects, right?

Reply:
Tried to mimic doctrine's `refresh` functionality. 
The `&` is not needed in case i'd modify passed object. In this code the `$entity` gets new object.